src/batch.ts

```diff
@@ -114,10 +126,37 @@ export class Batch {
      *   * "tMin": The time that each series reached its minimum
      *   * "tMax": The time that each series reached its maximum
      */
-    public extreme(name: keyof Extremes<SeriesSet>): SeriesSet {
+    public extreme(name: keyof Extremes<UserTypeSeriesSet>): UserTypeSeriesSet {
         return this.findExtremes()[name];
     }
 
+    /**
+     * Expands varying parameters as defined in BatchPars.varying into an array of UserTypes, each of which
+     * represents a single combination of values for the varying parameters. We will combine these with the base
+     * pars for each run.
+     */
+    private expandVaryingParams(varyingPars: VaryingPar[]): UserType[] {
```

Comment:
this could be a free function rather than a method, and then would be easier to put a few unit tests on. I'd not be surprised if we use this elsewhere (in this project or adapt to another) so might be worthwhile.